plugins/user-authenticators/ldap/src/main/java/org/apache/cloudstack/ldap/LdapManagerImpl.java

```diff
@@ -115,7 +124,7 @@ public void onPublishMessage(String senderAddress, String subject, Object args)
                     long domainId = account.getDomainId();
                     LdapTrustMapVO ldapTrustMapVO = _ldapTrustMapDao.findByAccount(domainId, account.getAccountId());
                     if (ldapTrustMapVO != null) {
-                        String msg = String.format("Removing link between LDAP: %s - type: %s and account: %s on domain: %s",
+                        String msg = String.format(REMOVING_LDAP_LINK_ON_DOMAIN,
                                 ldapTrustMapVO.getName(), ldapTrustMapVO.getType().name(), account.getAccountId(), domainId);
```

Comment:
And here we could invoke that method, something like: `String msg = generateRemoveMessage(ldapTrustMapVO, ...)`

Reply:
ah, ok. a bit overkill I think